src/Conference/ConferenceCall.jsx

```diff
@@ -0,0 +1,64 @@
+import React, { useState, useEffect } from "react";
+import { Switch, Route, useParams } from "react-router-dom";
+import ConferenceDetails from "./ConferenceDetails";
+import UpdateConference from "./UpdateConference";
+import { getConferenceById } from "../api.js";
+
+const ConferenceCall = () => {
+  const [conference, setConference] = useState(null);
+  const [updatedConference, setUpdatedConference] = useState(null);
+  const [isLoading, setIsLoading] = useState(true);
+  const [errorCaught, setErrorCaught] = useState(false);
+  const { id } = useParams();
+
+  const fetchData = async conferenceId => {
+    setIsLoading(true);
+
+    try {
+      const conference = await getConferenceById(conferenceId);
+      setConference(conference);
+      setUpdatedConference({
+        name: conference.name,
+        dateTime: conference.dateTime.substring(
```

Comment:
why do you need to manipulate the `dateTime` you get back from the API?

Reply:
Because the backend uses `YYYY-MM-DDTHH:MM:SSZ` and date time local seems to require `YYYY-MM-DDTHH:MM:SS` to work

Reply:
This works, in case this gets unwieldy in the future you could also introduce a library such as momentjs (https://momentjs.com/) to handle this but probably not needed here :)